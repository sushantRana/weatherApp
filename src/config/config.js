module.exports = {
  APPID: 'a23c219b61541dabd9bd6b7bb9d7ee0f',
  GoogleURL: 'https://maps.googleapis.com/maps/api/js?key=AIzaSyBP2_Okzm1JzT2T1IJNOswN2W1OXnOb77E&v=3.exp&libraries=geometry,drawing,places',
  weatherAPIURL: 'http://api.openweathermap.org/data/2.5/forecast?'
};
