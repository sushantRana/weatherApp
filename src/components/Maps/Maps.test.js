import React from 'react';
import { shallow } from 'enzyme';
import Mapsfunction from './Maps';
import { Marker } from "react-google-maps"
import { GoogleURL } from '../../config/config';

const props = {
  lat: '',
  lng: '',
  googleMapURL:GoogleURL,
  loadingElement: <div style={{ height: `100%` }} />,
  containerElement: <div style={{ height: `800px` }} />,
  mapElement: <div style={{ height: `100%` }} />,
  isMarkerShown: true,
  onMarkerClick: jest.fn()
}
const map = shallow(<Mapsfunction {...props} />);

it("map renders correctly", () => {
  expect(map).toMatchSnapshot();
});
