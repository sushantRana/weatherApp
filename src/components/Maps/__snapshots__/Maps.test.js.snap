// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`map renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <withScriptjs(withGoogleMap(Mapsfunction))
    containerElement={
        <div
            style={
                Object {
                    "height": "800px",
                  }
            }
        />
    }
    googleMapURL="https://maps.googleapis.com/maps/api/js?key=AIzaSyC4R6AN7SmujjPUIGKdyao2Kqitzr1kiRg&v=3.exp&libraries=geometry,drawing,places"
    isMarkerShown={true}
    lat=""
    lng=""
    loadingElement={
        <div
            style={
                Object {
                    "height": "100%",
                  }
            }
        />
    }
    mapElement={
        <div
            style={
                Object {
                    "height": "100%",
                  }
            }
        />
    }
    onMarkerClick={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "style": Object {
        "height": "100%",
      },
    },
    "ref": null,
    "rendered": null,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "style": Object {
          "height": "100%",
        },
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
